src/main/java/com/example/bookstoreapp/service/impl/BookServiceImpl.java

```diff
@@ -46,6 +47,11 @@ public List<BookDto> findAll() {
         return bookMapper.toDtos(bookList);
     }
 
+    @Override
+    public List<BookDto> getBooksByPage(Pageable pageable) {
+        return bookRepository.findAll(pageable).stream().map(bookMapper::toDto).toList();
```

Comment:
```suggestion
        return bookRepository.findAll(pageable)
                                              .stream()
                                              .map(bookMapper::toDto)
                                              .toList();
```
Use stream formatting